File: Chapter 1/SelectionSort.py
class SelectionSort:
    def selectionSort(self, A, n):
        # write code here
        for i in range(n):
            index = i
            for j in range(i+1, n):
                if A[j] < A[index]:
                    index = j
            A[i], A[index] = A[index], A[i]
        return A

File: Chapter 1/SelectionSort.cpp
class SelectionSort {
public:
    int* selectionSort(int* A, int n) {
        // write code here
        int index, temp;
        for(int i=0; i<n; i++){
            index = i;
            for(int j=i+1; j<n;j++){
                if(A[j] < A[index]){
                    index = j;
                }
            }
            temp = A[i];
            A[i] = A[index];
            A[index] = temp;
        }
        return A;
    }
};